composer.json

```diff
@@ -18,6 +18,9 @@
     "require": {
         "http-interop/http-factory": "^0.2"
     },
+    "require-dev": {
+        "phpunit/phpunit": "^5.7"
```

Comment:
This limits us to PHP 5.6+. I am okay with that, since PHP 5.5 is EOL, just wanted to have a note for future reference.